Samples/Windowing/cs-wpf/wpf_packaged_app/TitleBarPage.xaml.cs

```diff
@@ -82,34 +87,80 @@ private void TitlebarCustomBtn_Click(object sender, RoutedEventArgs e)
                 _mainAppWindow.TitleBar.ButtonPressedBackgroundColor = Colors.Green;
                 _mainAppWindow.TitleBar.ButtonPressedForegroundColor = Colors.White;
 
-                //Infer titlebar height
-                int titleBarHeight = _mainAppWindow.TitleBar.Height;
-                _mainWindow.MyTitleBar.Height = titleBarHeight;
```

Comment:
The titleBarHeight variable is only used here, please simplify:
_mainWindow.MyTitleBar.Height = _mainAppWindow.TitleBar.Height;

Reply:
Its used below as well

Reply:
Still feels unnecessary to create a local variable just to use the value from an already existing global. I'd consider updating both locations to use the appWindow.TitleBar.Height directly.